### ISSUE TITLE
add support for search in active files

### ISSUE THREAD

**Issue by not-lain** [googlecolab/colabtools#4271]
**Is your feature request related to a problem? Please describe.**
i want to search for a value in one of the files that i double clicked on in cloab

**Describe the solution you'd like**
a search for both the ipynb and the active file

**Describe alternatives you've considered**
* either add a search bar that searches both the ipynb file and the active file (if there's one that is open and active) 
* else add a search bar for the active file

**Additional context**

## this one shows 3
![image](https://github.com/googlecolab/colabtools/assets/70411813/2b5f8b0d-070d-4864-a71c-99aab3067d1e)

## after scrolling it shows 4
![image](https://github.com/googlecolab/colabtools/assets/70411813/ada997fe-60e2-4544-acb5-4736a1f24547)

**Comment by not-lain** [comment]
bump ! 
(the search feature is important) 
since I sometimes edit things manually on colab i find it kinda tedious to scroll 500+ lines looking for some x symbol when adding a small search button somewhere on the top right corner could solve this

TLDR ; 
we need a search button on open files

**Comment by wwwillchen** [comment]
Colab has a feature where you can search in your notebook (i.e. ipynb). Look at the screenshot below:

![findandreplace](https://github.com/googlecolab/colabtools/assets/7344640/c609f0c8-d067-4534-ac28-7d4204ae88cb)

Note: this doesn't support searching through your cell outputs (please let us know if this is your use case).

Hope that helps.

**Comment by not-lain** [comment]
@wwwillchen 
it doesn't work on the file viewer
![image](https://github.com/googlecolab/colabtools/assets/70411813/a669bc99-cb40-487e-8548-0e898ad6821b)

**Comment by not-lain** [comment]
the whole issue is all about support searches on file viewer, everything else works perfectly

**Comment by wwwillchen** [comment]
Got it - I misunderstood originally. Created an internal ticket to track this feature request: b/333725418

**Comment by not-lain** [comment]
@wwwillchen thank youuuuu ✨✨
